src/types/responses/ForeignAssets.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// This file is part of Substrate API Sidecar.
+//
+// Substrate API Sidecar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import { Option } from '@polkadot/types';
+import {
+	PalletAssetsAssetDetails,
+	PalletAssetsAssetMetadata,
+} from '@polkadot/types/lookup';
+
+import { IAt } from '.';
+
+export interface IForeignAssetInfo {
+	foreignAssetInfo: Option<PalletAssetsAssetDetails>;
+	foreignAssetMetadata: Option<PalletAssetsAssetMetadata>;
```

Comment:
This needs to be changed to just `AssetMetadata`, I would prefer if we also just use `AssetMetadata` here because its the generic type for an Asset where as `PalletAssetsAssetMetadata` is referring to the Asset Pallet.